notifications/notification_platform_bridge_notification_timeout_mac.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2019 The Brave Software Team. Distributed under the MPL2
```

Comment:
these should be under `brave/browser/notifications`

Reply:
Fixed